[2023-10-28T19:18:13.053-0500] {processor.py:157} INFO - Started process (PID=77439) to work on /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-10-28T19:18:13.054-0500] {processor.py:829} INFO - Processing file /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-10-28T19:18:13.054-0500] {logging_mixin.py:151} INFO - [2023-10-28T19:18:13.054-0500] {dagbag.py:539} INFO - Filling up the DagBag from /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-10-28T19:18:13.057-0500] {logging_mixin.py:151} INFO - [2023-10-28T19:18:13.056-0500] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-10-28T19:18:13.057-0500] {logging_mixin.py:151} INFO - [2023-10-28T19:18:13.057-0500] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-10-28T19:18:13.057-0500] {processor.py:841} WARNING - No viable dags retrieved from /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-10-28T19:18:13.067-0500] {processor.py:179} INFO - Processing /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.016 seconds
[2023-10-28T19:19:02.247-0500] {processor.py:157} INFO - Started process (PID=77553) to work on /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-10-28T19:19:02.288-0500] {processor.py:829} INFO - Processing file /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-10-28T19:19:02.290-0500] {logging_mixin.py:151} INFO - [2023-10-28T19:19:02.289-0500] {dagbag.py:539} INFO - Filling up the DagBag from /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-10-28T19:19:02.294-0500] {logging_mixin.py:151} INFO - [2023-10-28T19:19:02.293-0500] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-10-28T19:19:02.294-0500] {logging_mixin.py:151} INFO - [2023-10-28T19:19:02.294-0500] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-10-28T19:19:02.295-0500] {processor.py:841} WARNING - No viable dags retrieved from /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-10-28T19:19:02.302-0500] {processor.py:179} INFO - Processing /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.057 seconds
[2023-10-28T19:20:01.210-0500] {processor.py:157} INFO - Started process (PID=77940) to work on /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-10-28T19:20:01.250-0500] {processor.py:829} INFO - Processing file /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-10-28T19:20:01.252-0500] {logging_mixin.py:151} INFO - [2023-10-28T19:20:01.251-0500] {dagbag.py:539} INFO - Filling up the DagBag from /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-10-28T19:20:01.257-0500] {logging_mixin.py:151} INFO - [2023-10-28T19:20:01.256-0500] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-10-28T19:20:01.257-0500] {logging_mixin.py:151} INFO - [2023-10-28T19:20:01.257-0500] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-10-28T19:20:01.257-0500] {processor.py:841} WARNING - No viable dags retrieved from /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-10-28T19:20:01.265-0500] {processor.py:179} INFO - Processing /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.057 seconds
[2023-10-28T19:28:51.311-0500] {processor.py:157} INFO - Started process (PID=80679) to work on /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-10-28T19:28:51.313-0500] {processor.py:829} INFO - Processing file /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-10-28T19:28:51.314-0500] {logging_mixin.py:151} INFO - [2023-10-28T19:28:51.313-0500] {dagbag.py:539} INFO - Filling up the DagBag from /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-10-28T19:28:51.317-0500] {logging_mixin.py:151} INFO - [2023-10-28T19:28:51.316-0500] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-10-28T19:28:51.317-0500] {logging_mixin.py:151} INFO - [2023-10-28T19:28:51.317-0500] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-10-28T19:28:51.317-0500] {processor.py:841} WARNING - No viable dags retrieved from /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-10-28T19:28:51.326-0500] {processor.py:179} INFO - Processing /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2023-10-28T19:29:37.817-0500] {processor.py:157} INFO - Started process (PID=80832) to work on /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-10-28T19:29:37.857-0500] {processor.py:829} INFO - Processing file /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-10-28T19:29:37.859-0500] {logging_mixin.py:151} INFO - [2023-10-28T19:29:37.859-0500] {dagbag.py:539} INFO - Filling up the DagBag from /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-10-28T19:29:37.865-0500] {logging_mixin.py:151} INFO - [2023-10-28T19:29:37.864-0500] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-10-28T19:29:37.865-0500] {logging_mixin.py:151} INFO - [2023-10-28T19:29:37.865-0500] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-10-28T19:29:37.866-0500] {processor.py:841} WARNING - No viable dags retrieved from /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-10-28T19:29:37.874-0500] {processor.py:179} INFO - Processing /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.059 seconds
[2023-10-28T19:30:29.935-0500] {processor.py:157} INFO - Started process (PID=80955) to work on /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-10-28T19:30:29.974-0500] {processor.py:829} INFO - Processing file /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-10-28T19:30:29.976-0500] {logging_mixin.py:151} INFO - [2023-10-28T19:30:29.975-0500] {dagbag.py:539} INFO - Filling up the DagBag from /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-10-28T19:30:29.980-0500] {logging_mixin.py:151} INFO - [2023-10-28T19:30:29.979-0500] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-10-28T19:30:29.980-0500] {logging_mixin.py:151} INFO - [2023-10-28T19:30:29.980-0500] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-10-28T19:30:29.980-0500] {processor.py:841} WARNING - No viable dags retrieved from /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-10-28T19:30:29.989-0500] {processor.py:179} INFO - Processing /home/oscar_rincon/.pyenv/versions/3.10.6/envs/oscar_rincon/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.057 seconds
